test/Tool/too-many-phis.opt

```diff
@@ -1,1881 +0,0 @@
-%1788:i64 = xor %37, %1787
-%1789:i64 = xor %36, %1788
-%1790:i64 = xor %35, %1789
-%1791:i64 = mul 11376068507788127593:i64, %1790
-%1792:i64 = xor %1791, %35
-%1793:i64 = lshr %1791, 47:i64
-%1794:i64 = xor %1792, %1793
-%1795:i64 = mul 11376068507788127593:i64, %1794
-%1796:i64 = lshr %1795, 47:i64
-%1797:i64 = xor %1795, %1796
-%1798:i64 = mul 11376068507788127593:i64, %1797
-%1799:i64 = xor %31, %1798
-%1800:i64 = xor %30, %1799
-%1801:i64 = xor %29, %1800
-%1802:i64 = mul 11376068507788127593:i64, %1801
-%1803:i64 = xor %1802, %29
-%1804:i64 = lshr %1802, 47:i64
-%1805:i64 = xor %1803, %1804
-%1806:i64 = mul 11376068507788127593:i64, %1805
-%1807:i64 = lshr %1806, 47:i64
-%1808:i64 = xor %1806, %1807
-%1809:i64 = mul 11376068507788127593:i64, %1808
-%1810:i64 = xor %25, %1809
-%1811:i64 = xor %24, %1810
-%1812:i64 = xor %23, %1811
-%1813:i64 = mul 11376068507788127593:i64, %1812
-%1814:i64 = xor %1813, %23
-%1815:i64 = lshr %1813, 47:i64
-%1816:i64 = xor %1814, %1815
-%1817:i64 = mul 11376068507788127593:i64, %1816
-%1818:i64 = lshr %1817, 47:i64
-%1819:i64 = xor %1817, %1818
-%1820:i64 = mul 11376068507788127593:i64, %1819
-%1821:i64 = xor %19, %1820
-%1822:i64 = xor %18, %1821
-%1823:i64 = xor %17, %1822
-%1824:i64 = mul 11376068507788127593:i64, %1823
-%1825:i64 = xor %1824, %17
-%1826:i64 = lshr %1824, 47:i64
-%1827:i64 = xor %1825, %1826
-%1828:i64 = mul 11376068507788127593:i64, %1827
-%1829:i64 = lshr %1828, 47:i64
-%1830:i64 = xor %1828, %1829
-%1831:i64 = mul 11376068507788127593:i64, %1830
-%1832:i64 = xor %13, %1831
-%1833:i64 = xor %12, %1832
-%1834:i64 = xor %11, %1833
-%1835:i64 = mul 11376068507788127593:i64, %1834
-%1836:i64 = xor %1835, %11
-%1837:i64 = lshr %1835, 47:i64
-%1838:i64 = xor %1836, %1837
-%1839:i64 = mul 11376068507788127593:i64, %1838
-%1840:i64 = lshr %1839, 47:i64
-%1841:i64 = xor %1839, %1840
-%1842:i64 = mul 11376068507788127593:i64, %1841
-%1843:i64 = xor %7, %1842
-%1844:i64 = xor %6, %1843
-%1845:i64 = xor %5, %1844
-%1846:i64 = mul 11376068507788127593:i64, %1845
-%1847:i64 = xor %1846, %5
-%1848:i64 = lshr %1846, 47:i64
-%1849:i64 = xor %1847, %1848
-%1850:i64 = mul 11376068507788127593:i64, %1849
-%1851:i64 = lshr %1850, 47:i64
-%1852:i64 = xor %1850, %1851
-%1853:i64 = mul 11376068507788127593:i64, %1852
-%1854:i64 = xor %1, %1853
-%1855:i64 = var
-%1856:i64 = add 16:i64, %1855
-%1857:i64 = shl %1856, 48:i64
-%1858:i64 = lshr %1856, 16:i64
-%1859:i64 = or %1857, %1858
-%1860:i64 = var
-%1861:i64 = var
-%1862:i64 = xor %1860, %1861
-%1863:i64 = xor %1859, %1862
-%1864:i64 = mul 11376068507788127593:i64, %1863
-%1865:i64 = xor %1864, %1859
-%1866:i64 = lshr %1864, 47:i64
-%1867:i64 = xor %1865, %1866
-%1868:i64 = mul 11376068507788127593:i64, %1867
-%1869:i64 = lshr %1868, 47:i64
-%1870:i64 = xor %1868, %1869
-%1871:i64 = mul 11376068507788127593:i64, %1870
-%1872:i64 = xor %1855, %1871
-%1873:i64 = phi %0, %1854, %1872
-%1874:i64 = var
-cand %1873 %1874
```

Comment:
Out of curiosity, what's the reason for removing this test?

Reply:
The test was checking for "Value too large", but now it's not too large anymore returning "Invalid". Thus, it defeats the purpose of the test, no?